### ISSUE TITLE
Support turning groups on/off

### ISSUE THREAD

**Issue by w1ll1am23** [estevez-dev/ha_client#186]
Request to add support for toggling an entire group at one time.

**Comment by estevez-dev** [comment]
Do you mean a switch on card header? Or it is about a switch for group.all_automations, for example?

**Comment by w1ll1am23** [comment]
Well my use case is as follows.

I have a ceiling fan which takes two e12 bulbs. I only ever turn them on at the exact same time. I just recently moved from Wink to native Home Assistant ZWave/ZigBee. In Wink I didn't have a problem because they allow grouping lights which then show in Home Assistant as 1 light bulb. However now I get two light bulbs. I grouped those two light bulbs via Home Assistant groups and I am able to toggle them on/off via the web. 

What the group looks like in web (its called `Fan Light`)
<img width="497" alt="screen shot 2018-11-13 at 8 37 18 am" src="https://user-images.githubusercontent.com/6432770/48416971-8ec3d000-e71f-11e8-9271-d173367eaca5.png">


And via HA Client
![screenshot_ha_client_20181113-083646](https://user-images.githubusercontent.com/6432770/48417013-b3b84300-e71f-11e8-95f8-db2af7ba2cda.png)

![screenshot_ha_client_20181113-083651](https://user-images.githubusercontent.com/6432770/48417008-b024bc00-e71f-11e8-8145-5ec36c35ee04.png)
